### PR TITLE
docs:update languages/zh-CN/spec.md(#982)

### PR DESCRIPTION
**Pull request by sunshengjing**
update languages/zh-CN/spec.md
Signed-off-by:Shengjing SUN sunshengjing@gmail.com

### CODE REVIEW

**Review by lance — COMMENTED** [review]
I cannot contribute much feedback here, since I can't read most of the text. But scrolling through it, I did notice one English language typo.

File: cloudevents/languages/zh-CN/spec.md [hunk 1]
```diff
@@ -109,47 +103,43 @@ CloudEvents是一个以通用格式来描述事件数据的标准。它提供了
...
-#### 协议
+#### Protocol/协议
 
-消息可以通过各种行业标准协议
-（例如 HTTP、AMQP、MQTT、SMTP）、开源协议（例如 Kafka、NATS）或平台/供应商
+消息可以通过各种行业标准协议（例如 HTTP、AMQP、MQTT、SMTP）、开源协议（例如 Kafka、NATS）或平台/供应商
 专有协议（AWS Kinesis、Azure 事件网格）传输。
 
-#### 协议绑定
+#### Protocol Binkding/协议绑定
```
[the remark this answers is not in the document]
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md:117]: thanks for pointing it out!

File: cloudevents/languages/zh-CN/spec.md [hunk 2]
```diff
@@ -55,9 +50,9 @@ CloudEvents是一个以通用格式来描述事件数据的标准。它提供了
 
 ### 术语
 
-本规范定义了下列术语
+本规范定义了下列术语。
 
-#### Occur/发生
+#### Occurrence/发生
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/spec.md:55]:
```suggestion
#### Occurrence/事件发生
```

File: cloudevents/languages/zh-CN/spec.md [hunk 3]
```diff
@@ -55,9 +50,9 @@ CloudEvents是一个以通用格式来描述事件数据的标准。它提供了
...
 
 “发生”是指在软件系统运行期间对事实状态的捕获。
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/spec.md:57]:
```suggestion
“事件发生”是指在软件系统运行期间对事实状态的捕获。
```

File: cloudevents/languages/zh-CN/spec.md [hunk 4]
```diff
@@ -109,47 +103,43 @@ CloudEvents是一个以通用格式来描述事件数据的标准。它提供了
...
+### Naming Convention/命名约定
 
 CloudEvents 规范定义了到各种协议和编码的映射，随附的 CloudEvents SDK 面向各种运行时和编程语言。
-其中一些将元数据元素区分大小写，而另一些则不区分，
-并且单个 CloudEvent 可能通过涉及到协议、编码和运行时混合的多个跃点进行路由。
+其中一些将元数据元素区分大小写，而另一些则不区分，并且单个 CloudEvent 可能通过涉及到协议、编码和运行时混合的多个跃点进行路由。
 因此，本规范限制了所有属性的可用字符集，以防止区分大小写问题或与通用语言中标识符的合法字符集冲突问题。
 
-CloudEvents 属性名称必须由来自 ASCII 字符集的小写字母（“a”到“z”）或数字（“0”到“9”）组成。
+在跨传输协议和消息格式时，为了最大化互操作性和可移植性，CloudEvents 属性名称必须由来自 ASCII 字符集的小写字母（“a”到“z”）或数字（“0”到“9”）组成。
```
Comment (JieDing) [inline cloudevents/languages/zh-CN/spec.md:138]: Can you wrap words at 80 columns?

**Review by JieDing — COMMENTED** [review]
Thanks a ton for your hard works. I just left some suggestions.

File: cloudevents/languages/zh-CN/spec.md [hunk 5]
```diff
@@ -23,13 +19,13 @@ CloudEvents 是一个用于定义事件格式的供应商中立规范。
...
 对事件的统一描述的匮乏意味着开发者必须不断重新学习如何消费不同定义的事件。
-它同样限制了那些用来帮助事件数据完成 跨环境传输的库(如SDKs)，
+它同样限制了那些用来帮助事件数据完成跨环境传输的库(如SDKs)，
 工具(如事件路由器)和基础设施(如事件追踪系统)的发展。
 总体来看，这种匮乏严重阻碍了事件数据的可移植性和生产力。
 
 CloudEvents是一个以通用格式来描述事件数据的标准。它提供了事件在服务、平台和系统中的互操作性。
 
-事件格式指定如何使用某些编码格式序列化一个 CloudEvent。
+事件格式指定如何使用某些编码格式序列化一个CloudEvent。
```
[the remark this answers is not in the document]
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md:28]: ok, got it. I will review the rest, and revise all.

File: cloudevents/languages/zh-CN/spec.md [hunk 6]
```diff
@@ -109,47 +103,43 @@ CloudEvents是一个以通用格式来描述事件数据的标准。它提供了
...
-将事件直接映射到传输包的正文，或者可以为包提供额外的格式和结构。
+协议绑定可以选择使用[Event Format](#event-format),将事件直接映射到传输包的正文，或者可以为包提供额外的格式和结构。
 例如，可以使用结构化模式消息的包装器，或者可以将多个消息一起批处理到传输包正文中。
 
 ## Context Attributes/上下文属性
 
 每个符合本规范的 CloudEvent 必须包括指定为 REQUIRED 的上下文属性，
-可以包括一个或多个OPTIONAL上下文属性，
-并且可以包括一个或多个扩展属性。
+可以包括一个或多个OPTIONAL上下文属性，并且可以包括一个或多个扩展属性(#extension-context-attributes)。每个上下文属性只能在一个CloudEvent出现一次。本规范中定义的上下文属性（对标扩展上下文属性）称为“核心上下文属性”。
```
[the remark this answers is not in the document]
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md:127]: I did not catch it, can you explain in a bit more detail? thanks

[the remark this answers is not in the document]
Comment (JieDing) [reply cloudevents/languages/zh-CN/spec.md:127]: > I did not catch it, can you explain in a bit more detail? thanks

Wrap you words at 80 columns means that you write codes with line length no greater than 80 columns. Some modern IDEs like Visual Studio or IDEA have editor guidelines to help you do that.

[the remark this answers is not in the document]
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md:127]: I got it, thanks.

**Comment by JieDing** [comment]
@sunshengjing 
Thanks again for your translation.

@duglin 
LGTM, I think it's ready for group discussion.

File: cloudevents/spec.md [hunk 7]
```diff
@@ -536,7 +536,7 @@ in the event directly.
 ## Privacy and Security
 
 Interoperability is the primary driver behind this specification, enabling such
-behavior requires some information to be made available _in the clear_ resulting
+behavior requires some information to be made available in the clear resulting
```
Comment (duglin) [inline cloudevents/spec.md:539]: @sunshengjing was the removal of the underscore on-purpose or an accident?

Comment (sunshengjing) [reply cloudevents/spec.md:539]: it was an accident.

Comment (sunshengjing) [reply cloudevents/spec.md:539]: sorry

Comment (duglin) [reply cloudevents/spec.md:539]: no worries - can you add it back?

Comment (sunshengjing) [reply cloudevents/spec.md]: sure, I added it back. thanks

File: cloudevents/languages/zh-CN/spec.md (hunk 4 above)
Comment (duglin) [reply cloudevents/languages/zh-CN/spec.md:139]: did you want to fix these?

File: cloudevents/languages/zh-CN/spec.md [hunk 8]
```diff
@@ -109,47 +103,43 @@ CloudEvents是一个以通用格式来描述事件数据的标准。它提供了
...
-其中一些将元数据元素区分大小写，而另一些则不区分，
-并且单个 CloudEvent 可能通过涉及到协议、编码和运行时混合的多个跃点进行路由。
+其中一些将元数据元素区分大小写，而另一些则不区分，并且单个 CloudEvent 可能通过涉及到协议、编码和运行时混合的多个跃点进行路由。
 因此，本规范限制了所有属性的可用字符集，以防止区分大小写问题或与通用语言中标识符的合法字符集冲突问题。
 
-CloudEvents 属性名称必须由来自 ASCII 字符集的小写字母（“a”到“z”）或数字（“0”到“9”）组成。
+在跨传输协议和消息格式时，为了最大化互操作性和可移植性，CloudEvents 属性名称必须由来自 ASCII 字符集的小写字母（“a”到“z”）或数字（“0”到“9”）组成。
 属性名称应该是描述性的和简洁的，并且长度不应超过 20 个字符。
 
+CloudEvent 属性不能使用’data‘命名；因为它是为某些事件格式预留的。
```
[the remark this answers is not in the document]
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md]: fixed

File: cloudevents/languages/zh-CN/spec.md [hunk 9]
```diff
@@ -269,11 +257,11 @@ CloudEvents 协议绑定或事件格式实现同样必须能够
   - 必要的
   - 必须是非空字符串
 
-#### type
+#### type/类型
 
 - 类型: `String`
 - 描述: 该属性包含一个值，描述与原始事件相关的事件类型。
-  该属性通常用于路由、可观察性、策略实施等。其格式是生产者定义的，可能包括诸如 `type`版本之类的信息
+  该属性通常用于路由、可观察性、策略实施等。其格式是生产者定义的，可能包括诸如 `type`版本之类的信息.
```
[the remark this answers is not in the document]
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md]: fiexed

File: cloudevents/languages/zh-CN/spec.md [hunk 10]
```diff
@@ -45,8 +41,7 @@ CloudEvents是一个以通用格式来描述事件数据的标准。它提供了
 "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" 需要按照
 [RFC 2119](https://tools.ietf.org/html/rfc2119) 中的描述来理解。
 
-为清楚起见，
-当一个功能被标记为“OPTIONAL”时，这意味着消息的
+为清楚起见，当一个功能被标记为“OPTIONAL”时，这意味着消息的
```
[the remark this answers is not in the document]
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md]: fixed

File: cloudevents/languages/zh-CN/spec.md [hunk 11]
```diff
@@ -100,7 +94,7 @@ CloudEvents是一个以通用格式来描述事件数据的标准。它提供了
 
 #### Data/数据
 
-关于事件的特定域信息（即有效负载）。这可能包括有关事件的信息、有关已更改数据的详细信息等。
+关于"发生"的特定域信息（即有效负载)。这可能包括有关“发生”的信息、有关已更改数据的详细信息等。
```
[the remark this answers is not in the document]
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md]: fixed

File: cloudevents/languages/zh-CN/spec.md (hunk 3 above)
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md]: fixed

File: cloudevents/languages/zh-CN/spec.md (hunk 4 above)
Comment (sunshengjing) [reply cloudevents/languages/zh-CN/spec.md:139]: I think I fixed it, thanks

**Comment by duglin** [comment]
Please let me know when you're done with all edits.

**Comment by sunshengjing** [comment]
> Please let me know when you're done with all edits.

I have fixed all of them, thanks.

**Comment by duglin** [comment]
Approved on the 4/28 call

**Pull request merged**